src/shared/proto/radio_primitive.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+// We call this a RadioPrimitive and not just a "Primitive"
+// because the contents of this message are a 1-1 mapping
+// with the existing `RadioPrimitive` we send over radio
+// using the MRF-Dongle.
+message RadioPrimitive
```

Comment:
Didn't want to put too much throught into defining the optimal wifi primitive because
- we only need a few primitives, the rest are unused
- this will significantly change after the spline navigator is implemented
- this may significantly change based on how we chose to redo intent/primitive layer

So for now this is just a stop gap